.travis.yml

```diff
@@ -23,46 +28,35 @@ notifications:
 
 sudo: false
 
-services:
-  - mysql
-  - redis
-  - elasticsearch
-
 language: python
 
 cache:
-  - apt
   - pip
 
 env:
-  - REQUIREMENTS=lowest REXTRAS=docs,tests
-  - REQUIREMENTS=release REXTRAS=docs,tests
-  - REQUIREMENTS=devel REXTRAS=docs,tests
+  - REQUIREMENTS=lowest
```

Comment:
As it has a model layer I think we should add tests for PostgreSQL, MySQL and SQLite.